views/eApostilles/uploadFiles.ejs

```diff
@@ -144,6 +144,15 @@
                                 <%= uploadedFileData.length %> files were uploaded
                             <% } %>
                         </h2>
+                        <% if (filesToDelete > 0) { %>
+                            <p id="file-upload-error" class="govuk-error-message">
+                                <% if (filesToDelete === 1) { %>
+                                    Too many files uploaded. You need to delete 1 file
+                                <% } else { %>
+                                    Too many files uploaded. You need to delete <%= filesToDelete %> files
+                                <% } %>
```

Comment:
```suggestion
Too many files uploaded. You need to delete <%= filesToDelete %>  <%= filesToDelete === 1 ?  'file' : 'fieles' %>
```

Reply:
p.s. if there' is a lot of this type of code, you would probably be better off implementing a pluralisation or i18n library. 
then in your template you would just do `i18n("errors.tooManyFiles", numberOfFiles)`

Reply:
That's a good point. I'll create a ticket for that 👍